Submitted By:            Daniel Schreij <contact at dschreij at gmail dot com>
Date:                    2016-11-17
Initial Package Version: 2.0.0
Upstream Status:         Not reported (no repository or dev site to report at)
Origin:                  https://www.libsdl.org/projects/smpeg/release/smpeg2-2.0.0.tar.gz
Description:             Fixes memset not found error by including the appropriate windows.h header.

--- audio/MPEGaudio.cpp	2016-11-17 15:51:45.000000000 -0400
+++ audio/MPEGaudio.cpp	2016-11-17 15:59:06.000000000 -0400
@@ -26,6 +26,10 @@
 #include "MPEGaudio.h"
 #include "MPEGstream.h"
 
+#if defined(_WIN32)
+#include <windows.h>
+#endif
+
 MPEGaudio:: MPEGaudio(MPEGstream *stream, bool initSDL)
     : sdl_audio(initSDL)
     , mpeg(stream)
